classes/shortcodes.php

```diff
@@ -216,20 +252,28 @@ function do_shortcode( $args ) {
 					switch ( $tax_format ) {
 						case 'csv':
 						default:
-							return ! empty( $term_names ) ? $args['before'] . implode( ', ', $term_names ) . $args['after'] : '';
+							return ! empty( $term_names ) ? $args['before']
+							                                . implode( ', ',
+									$term_names ) . $args['after'] : '';
 							break;
 					}
 				}
 			}
 
 			// Look for postmeta with location_ prepended to it
-			if ( $value = get_post_meta( absint( $post->ID ), 'location_' . $atts['data'], true ) ) {
-				return str_replace( '%self%', $value, $atts['before'] . $value . $atts['after'] );
+			if ( $value = get_post_meta( absint( $post->ID ),
+				'location_' . $atts['data'], true )
+			) {
+				return str_replace( '%self%', $value,
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Opening parenthesis of a multi-line function call must be the last content on the line
* Only one argument is allowed per line in a multi-line function call

<!-- Comment by [SideCI](https://sideci.com) -->